draft-ietf-quic-recovery.md

```diff
@@ -501,7 +501,13 @@ and prevents a server from sending a 1-RTT packet on a PTO expiration before it
 has the keys to process an acknowledgement.
 
 When a PTO timer expires, the PTO period MUST be set to twice its current
-value. This exponential reduction in the sender's rate is important because
+value. The PTO period is set back to the original value upon receiving an
+acknowledgement for a non-Initial packet. The PTO timer is not decreased when
+the client receives an Initial ACK to ensure the client's anti-deadlock timer
```

Comment:
How about changing the text to something like: _The PTO period is set back to the original value when receiving an acknowledgement, if the endpoint knows that the peer has completed address validation. The PTO timer is not decreased while the peer is validating the address, to ensure the endpoint's anti-deadlock timer does not fire..._

I think that is our intent.

Reply:
Agreed, accepted with some small editorial changes.